dhash/module.py

```diff
@@ -45,6 +64,42 @@ def _in_repost_channel(self, message: nextcord.Message) -> bool:
     async def dhash(self, ctx):
         await utils.discord.send_help(ctx)
 
+    @commands.guild_only()
+    @commands.check(check.acl)
+    @dhash.group(name="regex")
+    async def dhash_regex(self, ctx):
+        await utils.discord.send_help(ctx)
+
+    @commands.check(check.acl)
+    @dhash_regex.command(name="unset")
+    async def dhash_regex_unset(self, ctx):
+        HashConfig.set("allowed_urls", None)
+        self.allowed_urls = None
+        await ctx.reply(_(ctx, "Regex successfuly unset."))
+
+    @commands.check(check.acl)
+    @dhash_regex.command(name="set")
+    async def dhash_regex_set(self, ctx, regex: str):
+        """Set regex for URL dedupe. Regex must be valid.
```

Comment:
"dedupe"? Please use valid english.

Reply:
The information "Regex must be valid." is unnecessary, it makes sense that it has to be valid.

Reply:
Fixed.